toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -274,3 +287,23 @@ func isToolchainPackage(rpmPath string, toolchainRPMs []string) bool {
 	}
 	return false
 }
+
+func prepRpmCloner(outDir string, disableUpstreamRepos bool) (cloner *rpmrepocloner.RpmRepoCloner, err error) {
```

Comment:
Can this function be moved to a generic location and exposed globally?

Reply:
Done. Now have `rpmrepocloner.ConstrcuctClonerWithNetwork()` in `rpmrepocloner.go`. I've updated the image fetcher to use it as well.